.github/scripts/generate_ci_workflows.py

```diff
@@ -247,7 +247,7 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
         test_runner_type=LINUX_CPU_TEST_RUNNER,
         on_pull_request=True,
         enable_doc_jobs=True,
-        num_test_shards=2,
+        num_test_shards=20,
```

Comment:
20 is a bit unrealistic, we don't run true auto-scaling so this actually may make queue times longer since we do a 1:1 scaling for `check_run` events